lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -174,15 +174,21 @@ def run!(verify_result: true, allow_protected_ns: false, prune: true)
         MSG
         @logger.summary.add_paragraph(ColorizedString.new(warning).yellow)
       end
+      ::StatsD.event("Deployment of #{@namespace} succeeded on #{@context}", 'Successfully deployed all resources.',
+        tags: statsd_tags << "status:success")
       ::StatsD.measure('all_resources.duration', StatsD.duration(start), tags: statsd_tags << "status:success")
       @logger.print_summary(:success)
     rescue DeploymentTimeoutError
       @logger.print_summary(:timed_out)
+      ::StatsD.event("Deployment of #{@namespace} timed out on #{@context}", 'Timed out while deploying all resources.',
```

Comment:
I find `'Timed out while deploying all resources.'` To be a bit odd grammatically, how about something like: 'One or more resources filed to deploy in time'